ui/src/treeDataProvider/AzureAccountTreeItemBase.ts

```diff
@@ -59,6 +61,8 @@ export abstract class AzureAccountTreeItemBase extends AzExtParentTreeItem imple
     }
 
     public async loadMoreChildrenImpl(_clearCache: boolean, context: types.IActionContext): Promise<AzExtTreeItem[]> {
+        // Refresh the AzureAccount, to handle the Azure account extension installed/uninstalled between refreshes
+        this._azureAccountTask = this.loadAzureAccount(this._testAccount);
```

Comment:
I believe this means `loadAzureAccount` will subscribe to some events multiple times. Can you double check and make sure we only subscribe to those events once

Reply:
Now I call the loadAzureAccount only when the old account is null.